e2e/test/scenarios/admin/datamodel/datamodel.cy.spec.js

```diff
@@ -67,8 +67,9 @@ describe("scenarios > admin > datamodel > field > field type", () => {
     });
   }
 
-  function searchFieldType(type) {
-    cy.findByPlaceholderText("Find...").type(type);
+  function searchFieldType(value) {
+    // .type() is flaky when used for ListSearchField - typed characters can sometimes get rearranged while typing
```

Comment:
what about adding some timeout during typing? e.g. 10ms?

Reply:
Yeah, that works too. BTW `delay` is 10ms by default.
I set it to 30.

Reply:
Unfortunately this is not enough - [the stress test fails](https://github.com/metabase/metabase/actions/runs/11704523659/job/32597146944).
This was the 2nd run. Interestingly 1st run passed 100/100.

I don't think increasing the delay is reliable here, unless we set a really large one.
So I think it's better to go back to original solution since it's faster. WDYT @uladzimirdev ?

Reply:
let's go with initial solution or with `paste`